quickstarts/pg-embedding/config.yml

```diff
@@ -0,0 +1,58 @@
+
+slug: postgres-embedding
+description: |
+  ## Why should you monitor your usage of Postgres Embedding?
+
+  Monitor your vector searches on Postgres Embedding to get visibility on what you send to Postgres Embedding, responses retrieved from Postgres Embedding, latency, usage and errors. 
+
+  ### Track the query performance of your Vector DB
+  Track the behavior of your vector stores. Monitor the latency, queries, the number of documents retrieved, and the content of the documents so that you can evaluate their relevance.
+ 
+  ### Track your app:  
+  By tracking key metrics like latency, throughput, error rates, and input & output, you can gain insights into your app's performance and identify areas of improvement.
+
+
+  ### What’s included in this quickstart?
+
+  New Relic LangChain monitoring quickstart provides a variety of pre-built dashboards, which will help you gain insights into the health and performance of your LangChain app. These reports include:
+  
+  - Vector searches
+  - Alerts for errors, search per vector store, and response time
+  - Identify popular queries, sources, and content
+
+summary: |
+  Monitor your Vector search's performance and quality with New Relic Postgres Embedding quickstart.
```

Comment:
```suggestion
  Monitor your Postgres Embedding's performance and quality with New Relic Postgres Embedding quickstart.
```

Reply:
Hi @rahulbasu-nr,
I have made changes based on your comments. Kindly review it.